api/open_saves.proto

```diff
@@ -342,6 +369,49 @@ message CreateBlobRequest {
   }
 }
 
+message CreateChunkedBlobRequest {
+  // store_key is the key of the store that the record belongs to.
+  string store_key = 1;
+
+  // record_key is the key of the record the new blob object belongs to.
+  string record_key = 2;
+
+  // Size of each chunk
+  int64 chunk_size = 3;
+}
+
+message CreateChunkedBlobResponse {
+  string session_id = 1;
+}
+
+message UploadChunkRequest {
+  oneof request {
+    ChunkMetadata metadata = 1;
+    bytes content = 2;
+  }
+}
+
+message UploadChunkResponse {
+  // TODO(yuryu): what do we need?
+}
+
+message ChunkMetadata {
+  bytes session_id = 1;
```

Comment:
I think I'm missing something regarding sessions. Why can we not just upload chunks based on a store/record ID + chunk number?